deploy/templates/nstemplatetiers/advanced/ns_code.yaml

```diff
@@ -20,7 +20,7 @@ objects:
   metadata:
     labels:
       toolchain.dev.openshift.com/provider: codeready-toolchain
-    name: toolchain-che-edit
+    name: rbac-edit
```

Comment:
what about `user-edit`? This would match with the rolebinding as suggested in https://github.com/codeready-toolchain/host-operator/pull/166/files#diff-0d5d074f131b625ee50490c7d955290aR53

Reply:
as I wrote, we already have RoleBinding `user-edit` that is used for different `ClusterRole`.